nucliadb/Makefile

```diff
@@ -0,0 +1,64 @@
+.PHONY: install-dev
```

Comment:
hmmm, this is actually a bit more of a problem to get it work correctly with pre commit because one of the values of pre-commit is that it passes a list of files that should be checked. I'm not sure of a nice way to do this with the make file right now

you have any ideas?

Reply:
I've restored old isort behavior for now until we can unscrew this some.